<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>428</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>665</width>
    <height>428</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alarm Widget [REMASTERED] - Debtanu Gupta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.needed/icons/alarm.png</normaloff>.needed/icons/alarm.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblAlarmName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Alarm Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpWeekRepeat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Week Repeat</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="chkSunday">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sunday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkMonday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Monday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkTuesday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkWednesday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkSaturday">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Saturday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkFriday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Friday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkThursday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Thursday</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCheckAll">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Check All</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUncheckAll">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Uncheck All</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblSpecialDates">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>68</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Special Date(s)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpecialDay">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>176</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Special Days</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpecialMonth">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>216</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Special Months</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblSpecialYear">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>256</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Special Years</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAlarmName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>9</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="leTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
    <property name="toolTip">
     <string>HH:MM AM/PM in 12 Hours Format</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teSpecialDates">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
    <property name="toolTip">
     <string>DD/MM/YYYY, DD/MM/YYYY-DD/MM/YYYY, ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnToday">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/today.png</normaloff>.needed/icons/today.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTomorrow">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tomorrow</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/tomorrow.png</normaloff>.needed/icons/tomorrow.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSpecialDays">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
    <property name="toolTip">
     <string>DD, DD-DD, ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSpecialMonths">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
    <property name="toolTip">
     <string>MM, MM-MM, ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leSpecialYears">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
     </font>
    </property>
    <property name="toolTip">
     <string>YYYY, YYYY-YYYY, ...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpAlarmRingtone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>411</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Alarm Ringtone</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>.needed/icons/play.png</normaloff>.needed/icons/play.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>.needed/icons/stop.png</normaloff>.needed/icons/stop.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBrowse">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mongolian Baiti</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Browse Ringtone...</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblRingtoneName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/save.png</normaloff>.needed/icons/save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mongolian Baiti</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>.needed/icons/view.png</normaloff>.needed/icons/view.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
